src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -0,0 +1,187 @@
+import math
+from collections import defaultdict
+from datetime import datetime
+from typing import cast
+
+import sentry_sdk
+from django.utils import timezone
+from sentry_sdk import set_tag
+
+from sentry import features
+from sentry.constants import DataCategory
+from sentry.models.activity import Activity
+from sentry.models.group import Group
+from sentry.models.organization import Organization, OrganizationStatus
+from sentry.models.organizationmember import OrganizationMember
+from sentry.models.release import Release, ReleaseProject
+from sentry.services.hybrid_cloud.user_option import user_option_service
+from sentry.silo import SiloMode
+from sentry.snuba.referrer import Referrer
+from sentry.tasks.base import instrumented_task, retry
+from sentry.tasks.summaries.utils import (
+    ONE_DAY,
+    DailySummaryProjectContext,
+    OrganizationReportContext,
+    fetch_key_error_groups,
+    fetch_key_performance_issue_groups,
+    project_event_counts_for_organization,
+    project_key_errors,
+    project_key_performance_issues,
+    user_project_ownership,
+)
+from sentry.types.activity import ActivityType
+from sentry.types.group import GroupSubStatus
+from sentry.utils.dates import floor_to_utc_day, to_datetime, to_timestamp
+from sentry.utils.outcomes import Outcome
+from sentry.utils.query import RangeQuerySetWrapper
+
+
+# The entry point. This task is scheduled to run every day at 4pm PST.
+@instrumented_task(
+    name="sentry.tasks.daily_summary.schedule_organizations",
+    queue="reports.prepare",
+    max_retries=5,
+    acks_late=True,
+    silo_mode=SiloMode.REGION,
+)
+@retry
+def schedule_organizations(timestamp: float | None = None, duration: int | None = None) -> None:
+    if timestamp is None:
+        # The time that the report was generated
+        timestamp = to_timestamp(floor_to_utc_day(timezone.now()))
+
+    if duration is None:
+        # The total timespan that the task covers
+        duration = ONE_DAY
+
+    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
+    for organization in RangeQuerySetWrapper(
+        organizations, step=10000, result_value_getter=lambda item: item.id
+    ):
+        if features.has("organizations:daily-summary", organization):
```

Comment:
daily-summaries are an org flag and not a user flag? 🤔

Reply:
Why not? For the MVP we want to opt in the Sentry org to see if people find it useful.